src/utils/api.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+
+async function getAllProjects() {
+    let response = await axios.get('https://projecthunt-api.herokuapp.com/project')
+        .catch(error => { console.log(error) })
+
+    return response;
+}
+
+function createProject(project) {
+    axios.post('https://projecthunt-api.herokuapp.com/project', project)
+        .then(response => console.log(response))
+        .catch(error => console.log(error))
+}
+
```

Comment:
pls, maintain es6 standards. You can use the arrow function instead of this function.